templates/app/webpack-default/template/src/home.js

```diff
@@ -0,0 +1,34 @@
+import { createWindow, createTab, faIcon } from './utils';
+
+const title = 'Home';
+const window = createWindow({ title });
+
+const wrapper = Ti.UI.createView({ layout: 'vertical', height: Ti.UI.SIZE });
+const searchIcon = faIcon('edit', 96, { color: '#ccc' });
+wrapper.add(searchIcon);
+const hint = Ti.UI.createLabel({
+	attributedString: Ti.UI.createAttributedString({
+		text: 'Edit home.js to change this view!',
+		attributes: [ {
+			type: Ti.UI.ATTRIBUTE_FOREGROUND_COLOR,
+			value: '#E82C2A',
+			range: [ 5, 7 ]
```

Comment:
totally unrelated to this PR, but this syntax really confuses me for range. I guess since I'm used to mathematical ranges being declared like `(0, 5]` or `[1, 6]` or `[1, 6)` where `(` is exclusive and `[` is inclusive. Obviously this is JS syntax and an array, but my brain immediately thinks: "oh, he's selecting text indices 5 to 7 inclusive" when actually you're selecting text starting at index 5, but for the next 7 characters. Not sure if anyone has any thoughts on making this more obvious without breaking backwards compatibility....
cc @jquick-axway @ewanharris @garymathews @vijaysingh-axway

Reply:
I can totally relate. While writing this i was like, what the hell why is this range not doing what i want? This does not make any sense! Until i eventually looked up the docs and found out that the second value is the length, and not the end of the range.

Reply:
We could modify the "range" property to accept either a dictionary or array. With a dictionary, you effectively have named parameters, making it more obvious how it works.
```javascript
var attributedString = Ti.UI.createAttributedString({
	text: "Hello World",
	attributes: [{
		type: Ti.UI.ATTRIBUTE_UNDERLINES_STYLE,
		value: Ti.UI.ATTRIBUTE_UNDERLINE_STYLE_SINGLE,
		range: {  // Add support for dictionary.
			index: 0,
			length: 5,
		},
	}, {
		type: Ti.UI.ATTRIBUTE_BACKGROUND_COLOR,
		value: "red",
		range: [0, 5],  // <- The old way.
	}],
});
```

Reply:
Just another +1 on "wtf does that syntax mean". I like Josh's suggestion, it makes it a lot clearer to me

Reply:
The dictionary seems like a great idea. I would prefer `{ start: 6, length: 5 }` though ;)